superflore/utils.py

```diff
@@ -199,8 +199,8 @@ def get_license(l):
     elif re.search(pub_dom_re, l, f):
         return 'public_domain'
     else:
-        err('Could not match license "{0}".'.format(l))
-        raise UnknownLicense('bad license')
+        warn('Could not match license "{0}". Passing it through...'.format(l))
```

Comment:
This change requires some test changes. Please modify the appropriate tests.

Reply:
Done